python/cuspatial/cuspatial/core/binops/distance_dispatch.py

```diff
@@ -185,10 +185,10 @@ def __call__(self):
 
         # Rows with misaligned indices contains nan. Here we scatter the
         # distance values to the correct indices.
-        result = full(
-            len(self._res_index),
-            float("nan"),
+        result = as_column(
+            [float("nan")] * len(self._res_index),
```

Comment:
```suggestion
        result = as_column(
            float("nan"),
            length=len(self._res_index),
```

?

Reply:
That was tested in 4b6cb9f. See CI results:

```
FAILED python/cuspatial/cuspatial/tests/spatial/distance/test_distance.py::test_geoseires_distance_empty - AssertionError: Series are different

Series length are different
[left]:  1, RangeIndex(start=0, stop=1, step=1)
[right]: 0, RangeIndex(start=0, stop=0, step=1)
```

Reply:
With this diff against trunk:
```
diff --git a/python/cuspatial/cuspatial/core/binops/distance_dispatch.py b/python/cuspatial/cuspatial/core/binops/distance_dispatch.py
index c1995310..db2e6339 100644
--- a/python/cuspatial/cuspatial/core/binops/distance_dispatch.py
+++ b/python/cuspatial/cuspatial/core/binops/distance_dispatch.py
@@ -1,5 +1,5 @@
 import cudf
-from cudf.core.column import as_column, full
+from cudf.core.column import as_column
 
 from cuspatial._lib.distance import (
     pairwise_linestring_distance,
@@ -185,10 +185,11 @@ class DistanceDispatch:
 
         # Rows with misaligned indices contains nan. Here we scatter the
         # distance values to the correct indices.
-        result = full(
-            len(self._res_index),
+        result = as_column(
             float("nan"),
+            length=len(self._res_index),
             dtype="float64",
+            nan_as_null=False,
         )
         scatter_map = as_column(
             range(len(self._res_index)), dtype="int32"
```

I cannot replicate this error locally.